storage/innobase/buf/buf0flu.cc

```diff
@@ -2630,36 +2624,34 @@ buf_get_total_LRU_list_length(void)
 /*********************************************************************//**
 Adjust the desired page cleaner thread sleep time for LRU flushes.  */
 MY_ATTRIBUTE((nonnull))
-static
-void
+static void
 page_cleaner_adapt_lru_sleep_time(
-/*==============================*/
-	ulint*	lru_sleep_time,	/*!< in/out: desired page cleaner thread sleep
-				time for LRU flushes  */
-	ulint	lru_n_flushed) /*!< in: number of flushed in previous batch */
-
+    ut_monotonic_time
+	* lru_sleep_time, /*!< in/out: desired page cleaner thread
+                                      sleep time for LRU flushes  */
+    ulint lru_n_flushed)  /*!< in: number of flushed in previous batch */
 {
-        ulint free_len = buf_get_total_free_list_length();
-        ulint max_free_len = ut_min(buf_get_total_LRU_list_length(),
-                        srv_LRU_scan_depth * srv_buf_pool_instances);
+	ulint free_len = buf_get_total_free_list_length();
```

Comment:
Should not have been touched

Reply:
Hmm, odd, not sure why that changed unless clang-format picked it up and changed it as being adjacent/part of the diff from the signature change :-\